bsdlib/CHANGELOG.rst

```diff
@@ -5,6 +5,17 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+bsdlib 0.3.4
+************
+
+Updated library with various changes:
+
+* Added modem log interface (``NRF_PROTO_MODEM_LOG``).
+* Increased ``BSD_MAX_SOCKET_COUNT`` from 8 to 9.
+* Do not return error from :cpp:func:`nrf_inbuilt_key_exists()` if key does not exist.
```

Comment:
This line would need to be edited to match the style of other lines in this section.
```suggestion
Modified :cpp:func:`nrf_inbuilt_key_exists()` so that it does not return an error if a key does not exist.
```
Also, add information about what it does if a key does exist, even if it might be an obvious information.